admin_manual/maintenance/upgrade.rst

```diff
@@ -144,35 +144,35 @@ To do this, run the following command:
    
    sudo service apache2 stop
 
-Get a Copy of ownCloud 10.0.2
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Get a Copy of ownCloud |version|
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 You can download ownCloud 10 from several places. 
-However, the best place is <download.owncloud.org>. 
+However, the best place is https://owncloud.org/install/. 
 To do so, run the following command:
 
 .. code-block:: console
    
-   # Download and extract the code to /var/www/owncloud-10.0.2
-   wget -qO- https://download.owncloud.org/community/owncloud-10.0.2.tar.bz2 | \ 
-     tar --transform 's/^owncloud/owncloud-10.0.2/' -jxv -C /var/www/
+   # Download and extract the code to /var/www/owncloud-10.0.4
+   wget -qO- https://download.owncloud.org/community/owncloud-10.0.4.tar.bz2 | \ 
+     tar --transform 's/^owncloud/owncloud-10.0.4/' -jxv -C /var/www/
 
 Copy config/config.php to the New Source
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Next, copy ``config/config.php`` from the current installation to the new, 10.0.2, source.
+Next, copy ``config/config.php`` from the current installation to the new, |version|, source.
 You can do this by running the following command:
 
 .. code-block:: console
    
-   cp -v /var/www/owncloud/config/config.php /var/www/owncloud-10.0.2/config/config.php
+   cp -v /var/www/owncloud/config/config.php /var/www/owncloud-10.0.4/config/config.php
 
 If you use the default ownCloud data directory, then you need to copy it from the existing installation to the new source directory as well. 
 You can do this by running the following command:
 
 .. code-block:: console
    
-   cp -rv /var/www/owncloud/data /var/www/owncloud-10.0.2/data
+   cp -rv /var/www/owncloud/data /var/www/owncloud-|version|/data
```

Comment:
Why can substituting "|version|" into a path string work here, but not at e.g. line 168?
(and other command examples that have the version in the path string)

Reply:
I'll double-check that that works.

Reply:
Nope, it's broken. Will fix.